### PR TITLE
add CANON EOS M10 support (fixes #11006)

### PR DESCRIPTION
**Pull request by schenlap**

### CODE REVIEW

File: src/external/rawspeed/data/cameras.xml [hunk 1]
```diff
@@ -866,6 +866,24 @@
...
 	</Camera>
+	<Camera make="Canon" model="Canon EOS M10">
+		<ID make="Canon" model="EOS M10">Canon EOS M10</ID>
+		<CFA width="2" height="2">
+			<Color x="0" y="0">RED</Color>
+			<Color x="1" y="0">GREEN</Color>
+			<Color x="0" y="1">GREEN</Color>
+			<Color x="1" y="1">BLUE</Color>
+		</CFA>
+		<Crop x="72" y="50" width="0" height="-2"/>
```
Comment (LebedevRI) [inline src/external/rawspeed/data/cameras.xml]: I think this one still have 1 or 2 garbled rows.
I.e. try `y="52"`

**Comment by schenlap** [comment]
Maybe a little fuzzy. I changed it to 52. Better?

**Comment by LebedevRI** [comment]
@schenlap thank you.

I'm noticing that there is no wb presets.
Do you know about [tools/extract_wb](tools/extract_wb) ?
For many models, even any single raw contains all the wb presets.

**Comment by schenlap** [comment]
Yes, I konw extract_wb. But I did not know that one image can contain all wb presets, so I added wb_presets only if requested in the issue and files attached for every preset. I will add wb_presets as EOS M10 contains all preset in CR2 file :-)

**Pull request merged**